File: Modul-5/Soal 4/PRAK504-2410817110004-MuhammadNaufalKhalish.py
def reverse(a) :
    balik=0
    while a!=0:
        angkaterakhir=a%10
        balik=balik*10+angkaterakhir
        a=a//10
    return balik



A,B=map(int,input().split())
A=reverse(A)
B=reverse(B)
C=A+B
print(f"{reverse(C)}")

File: Modul-5/Soal 4/PRAK504-2410817110004-MuhammadNaufalKhalish.c
#include <stdio.h>
int reverse(int a){
    int balik=0;
    while(a!=0){
        
        int angkaterakhir=a % 10;
        balik=balik*10+angkaterakhir;
        a=a/10;
    }
    return balik;
}

int main(){
    int A,B;
    scanf("%d %d",&A, &B);
    A=reverse(A);
    B=reverse(B);
    int C=A+B;
    printf ("%d", reverse(C));


}